WFInfo/verifyCount.xaml.cs

```diff
@@ -42,22 +42,34 @@ public static void ShowVerifyCount( List<InventoryItem> itemList)
 
         private void SaveClick(object sender, RoutedEventArgs e)
         {
+            bool saveFailed = false;
             foreach (InventoryItem item in latestSnap)
             {
                 if (item.Name.Contains("Prime"))
                 {
                     string[] nameParts = item.Name.Split(new string[] { "Prime" }, 2, StringSplitOptions.None);
                     string primeName = nameParts[0] + "Prime";
-                    string partName = primeName + ( nameParts[1].Length > 10 ? nameParts[1].Replace(" Blueprint", "") : nameParts[1]);
+                    string partName = primeName + ( ( nameParts[1].Length > 10 && !nameParts[1].Contains("Kubrow") ) ? nameParts[1].Replace(" Blueprint", "") : nameParts[1]);
 
-                    Console.WriteLine(item.Name);
-                    Console.WriteLine(primeName);
-                    Console.WriteLine(partName);
-                    Main.dataBase.equipmentData[primeName]["parts"][partName]["owned"] = item.Count;
+                    Main.AddLog("Saving count \"" + item.Count + "\" for part \"" + partName + "\"");
+                    try
+                    {
+                        Main.dataBase.equipmentData[primeName]["parts"][partName]["owned"] = item.Count;
```

Comment:
Does this cause any issues for equipment screen when count is above maximum?

Reply:
In #175 I reworked the owned count to not be capped by how many is needed for a set. The equipment screen is fully capable of adding/removing items regardless of how many are needed for the set, obviously not allowing negative values. The owned/needed display for the set in the equipment screen is capped so a component can only contribute up to the required amount of that piece for the set.

The reason for the try/catch block there is to make sure it'll only fail with the one item that was an issue instead of full on crashing if it fails to save, like it did with kavasa prime bp without this PR